android/app/src/main/res/layout/fragment_schedule.xml

```diff
@@ -61,17 +61,5 @@
             android:background="@android:color/white"
             android:layout_below="@+id/filter_bar"/>
 
-        <android.support.design.widget.FloatingActionButton
-            android:id="@+id/schedule_fab_filter"
-            android:layout_width="@dimen/fab_width"
-            android:layout_height="@dimen/fab_height"
-            android:layout_alignParentBottom="true"
-            android:layout_alignParentEnd="true"
-            android:layout_alignParentRight="true"
-            android:layout_marginBottom="@dimen/layout_margin_large"
-            android:layout_marginEnd="@dimen/layout_margin_large"
-            android:layout_marginRight="@dimen/layout_margin_large"
-            app:srcCompat="@drawable/ic_filter_24dp"/>
```

Comment:
All problems are solved if you don't move this from fragment to activity and use fragment view to show snackbar instead

Reply:
@iamareebjamal  The reason why I moved it to the activity is because coordinate layout was used there and the fab moves up automatically in that layout. I am not able to do the same thing in fragment because the snackBar messages are given from the SessionsListAdapter. Any suggestions ?

Reply:
You can use CoordinatorLayout in Fragment itself and use that view to show snackbar

Reply:
` View view = LayoutInflater.from(context).inflate(R.layout.fragment_schedule, null);
        CoordinatorLayout coordinatorLayout = (CoordinatorLayout)view.findViewById(R.id.coordinate_layout_fab_filter);`

@iamareebjamal After writing this in the DayScheduleViewHolder I pass this coordinatorLayout in the snackBar but the snackBar messages are not displayed.
Can you please tell me how should I solve this
Thank you.

Reply:
Can you check if you have called `show()`?

Also, if you don't want, you can not add coordinatorlayout in fragment and use the one in activity by `getActivity().findViewById` in `onCreateView` method

Reply:
yes show() was called
@iamareebjamal this is exactly what I have done in the current PR and it works perfectly. Can you please see
Thank you

Reply:
You have also moved the FAB from fragment to activity

Reply:
@iamareebjamal I need the coordinator layout in the DayScheduleViewHolder because the snackbar messages are given from there so I can't use `getActivity().findViewById`. I don't why even after inflating the layout correctly ,the snackBar messages are not displayed.

Is it a wrong practice to store the fragment's FAB in the activity?
Thank you

Reply:
You can store the fab in Activity, but then show the snackbar using activity, and not fragment. So use the callback communication mechanism from fragment to activity